### PR TITLE
Replace deprecated command in url enrichment v2

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-4548](https://jira-hq.paloaltonetworks.local/browse/CIAC-4548)

## Description
replaced **vt-private-get-url-report** automation execution with vt URL automation execution 

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Review by idovandijk — CHANGES_REQUESTED** [review]
We shouldn't use `url` explicity in the enrichment playbooks, as discussed let's make it skippable instead and open a ticket to create a replacement

**Comment by YuvHayun** [comment]
> We shouldn't use `url` explicity in the enrichment playbooks, as discussed let's make it skippable instead and open a ticket to create a replacement

A ticket was already created (and attached), after consulting @bakatzir, we decided to replace the current task with a VT v3 task.

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_10.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### URL Enrichment - Generic v2
+- Note: removed VirusTotal private api tasks from playbook, it's recommended to use this playbook with VirusTotal v3 now.
```
Comment (JudahSchwartz) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_10.md:4]:
```suggestion
- Note: removed VirusTotal private api tasks from playbook. It is now recommended to use this playbook with VirusTotal v3.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-URL_Enrichment_-_Generic_v2_README.md [hunk 2]
```diff
@@ -40,9 +39,9 @@ This playbook does not use any sub-playbooks.
...
 | URL.Malicious | Whether the detected URL was malicious. | unknown |
-| URL.Vendor | The vendor that labeled the URL as malicious. | string |
-| URL.Description | The additional information for the URL. | string |
+| URL.Vendor | Vendor that labeled the URL as malicious. | string |
+| URL.Description | Additional information for the URL. | string |
 
 ## Playbook Image
 ---
-![URL_Enrichment_Generic_v2](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/URL_Enrichment_Generic_v2.png)
+![URL Enrichment - Generic v2](../doc_files/URL_Enrichment_-_Generic_v2.png)
```
Comment (JudahSchwartz) [inline Packs/CommonPlaybooks/Playbooks/playbook-URL_Enrichment_-_Generic_v2_README.md:47]: @MLainer1  shouldnt your validation for relative urls fail on this?

Comment (ostolero) [reply Packs/CommonPlaybooks/Playbooks/playbook-URL_Enrichment_-_Generic_v2_README.md:47]: I think it did :) see here the build on the earlier commit - https://app.circleci.com/pipelines/github/demisto/content/206781/workflows/19fd72b1-a561-47cb-9b00-fe735b1210ac/jobs/597577

**Review by idovandijk — APPROVED** [review]
nice

**Comment by YuvHayun** [comment]
@ShahafBenYakir Failing since I moved the playbook image to the specific pack's docs file. need force merge.

**Pull request merged**